config/v1/types_cluster_operator.go

```diff
@@ -188,6 +188,14 @@ const (
 	// allow updates when this condition is not False, including when it is
 	// missing, True, or Unknown.
 	OperatorUpgradeable ClusterStatusConditionType = "Upgradeable"
+
+	// ClusterFleetEvaluation is used to indicate the result of the detection
+	// logic that was added to a component to evaluate the introduction of an
+	// invasive change that could potentially result in highly visible alerts,
+	// breakages or upgrade failures on user  clusters. You can concatenate
+	// multiple ConditionStatus/Reason if you need to evaluation the
+	// introduction of multiple changes.
+	ClusterFleetEvaluation ClusterStatusConditionType = "ClusterFleetEvaluation"
```

Comment:
so is the idea that a component would set this to "true" when they detect one or more conditions they are investigating?

The condition name doesn't naturally lend itself to taking on a true/false value like our other conditions do.

I don't know how wordy we want to get, but "ClusterFleetEvaluationConditionPresent" or Detected would work better with a true/false value.

Reply:
I see your point. `EvaluationConditionDetected` might be more appropriate though I wonder if it should be singular or plural given there could be multiple conditions.

Reply:
`EvaluationConditionsDetected` works for me.

Reply:
@deads2k what is the delimiter for concatenating multiple Reasons? I think adding that to the doc will help developers.

Reply:
I have renamed the condition to `EvaluationConditionsDetected`

Reply:
> @deads2k what is the delimiter for concatenating multiple Reasons? I think adding that to the doc will help developers.

The delimiter is `::` from https://github.com/openshift/library-go/blob/master/pkg/operator/status/condition.go#L154

Reply:
Addressed @deads2k. Please review.